### ISSUE TITLE
loss is too high

### ISSUE THREAD

**Issue by zhouyuan888888** [scaelles/OSVOS-TensorFlow#27]
I come across a problem and very confused:

2019-02-23 22:36:17.311281 Iter 586: Training Loss = 79458.1719
2019-02-23 22:36:19.815631 Iter 588: Training Loss = 37970.0039
2019-02-23 22:36:22.456881 Iter 590: Training Loss = 46907.9766
2019-02-23 22:36:24.121243 Iter 592: Training Loss = 21860.0176
2019-02-23 22:36:25.718174 Iter 594: Training Loss = 49590.5156
2019-02-23 22:36:27.451493 Iter 596: Training Loss = 90751.0938
2019-02-23 22:36:29.451918 Iter 598: Training Loss = 17363.9922
2019-02-23 22:36:31.202461 Iter 600: Training Loss = 27310.3105
2019-02-23 22:36:33.080727 Iter 602: Training Loss = 81702.3984
2019-02-23 22:36:35.805301 Iter 604: Training Loss = 43151.9375
2019-02-23 22:36:38.184319 Iter 606: Training Loss = 26587.7422
2019-02-23 22:36:40.321566 Iter 608: Training Loss = 214049.2031
2019-02-23 22:36:42.982664 Iter 610: Training Loss = 11704.9385
2019-02-23 22:36:44.731441 Iter 612: Training Loss = 173234.9844
2019-02-23 22:36:46.219128 Iter 614: Training Loss = 1401.4938
2019-02-23 22:36:47.852391 Iter 616: Training Loss = 52776.4648
2019-02-23 22:36:49.122615 Iter 618: Training Loss = 31279.1348
2019-02-23 22:36:52.050488 Iter 620: Training Loss = 12059.2002
2019-02-23 22:36:53.660343 Iter 622: Training Loss = 67013.9453
2019-02-23 22:36:55.264316 Iter 624: Training Loss = 12743.4521
2019-02-23 22:36:57.738322 Iter 626: Training Loss = 10916.0781
2019-02-23 22:36:59.682404 Iter 628: Training Loss = 66693.2812

can you tell me why my loss is so high?can you tell me  your loss in your training~
                                                                           thank you so much!!!

**Comment by kmaninis** [comment]
It probably has to do with your image size. We do not divide the loss by the number of elements in the prediction, thus the high number. Is your image size comparable to ours (480p)?
